components/EmailLogin.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { useState } from 'react'
+
+export default function EmailLogin({onEmailSubmit, disabled}) {
+    const [email,setEmail] = useState('');
+
+
+    const handleSubmit = async(e: { preventDefault: () => void; }) => {
+        e.preventDefault();
+        onEmailSubmit(email);
+    }
+
+    return (
+        <>
+        <form onSubmit={handleSubmit} className="border rounded p-3 login-form">
+            <h2 className="text-center fs-2 mb-3">Login</h2>
+            <div className="mb-3">
+              <input
+                placeholder="Email Address"
+                className="form-control"
+                type="email"
+                value={email}
+                onChange={(e) => setEmail(e.target.value)}
+                required
+              />
+              {/*   {userEmail.meta.touched && userEmail.meta.error && (
+    <span>{userEmail.meta.error}</span>
```

Comment:
Can the commented out code in this file be removed?

Reply:
Yup. They were part of the original code that use react-hook-forms. I have already removed them from the code.